### PR TITLE
fix multipart form data implementation and broken tests

### PR DESCRIPTION
**Pull request by fgrutsch**
This was done together with @zZKato

This fixes the missing things for issue #65  and PR #291

### CODE REVIEW

File: server/play-server/src/main/scala/sttp/tapir/server/play/PlayServerOptions.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ case class PlayServerOptions(
     playBodyParsers: PlayBodyParsers
 ) {
 
-  lazy val logger: Logger = Logger(getClass.getPackageName)
+  lazy val logger: Logger = Logger("sttp.tapir.server.play")
```
Comment (ghostbuster91) [inline server/play-server/src/main/scala/sttp/tapir/server/play/PlayServerOptions.scala]: How about `sttp.tapir.server.play.getClass` ?

Comment (ghostbuster91) [reply server/play-server/src/main/scala/sttp/tapir/server/play/PlayServerOptions.scala]: Ok, I see, it doesn't work as well :(

Comment (ghostbuster91) [reply server/play-server/src/main/scala/sttp/tapir/server/play/PlayServerOptions.scala]: Maybe `this.getClass.getPackage.getName` ?

Comment (zZKato) [reply server/play-server/src/main/scala/sttp/tapir/server/play/PlayServerOptions.scala]: @ghostbuster91 updated

**Pull request merged**